### ISSUE TITLE
Panning in visual editor doesn't work on mobile

### ISSUE THREAD

**Issue by Mearman** [breadboard-ai/breadboard#2466]

**Comment by paullewis** [comment]
Yeah, I think in general we need to do a bit of a "mobile audit" because the Visual Editor isn't really geared towards it today. I'm not entirely sure anything smaller than – say – a tablet is all that helpful, if I'm candid.

**Comment by Mearman** [comment]
I think the current UI would be suitable for tablets and possibly square-ish soldering displays.
I was planning on doing a more focused pass-over of the UI. When doing so, I will consider touch, tablet, and mobile.
